docs/embedded/tutorials/register-ct.md

```diff
@@ -1,13 +1,62 @@
 ---
-title: Register Container Types
+title: Create and Register Container Types
 description: This article explains how to register a new Container Types work.
 ms.date: 11/28/2023
 ms.localizationpriority: high
 ---
 
-# Registering a Container Type
+# Create and Register a Container Type
 
-## Create a new Container Type for your app
+This article will explain how to create and register a billable [Container Type](../concepts/app-concepts/containertypes.md) in SharePoint Embedded.
+
+**Pre-requisites:**
+* You must have admin credentials to a M365 developer tenant
+* You must register your application in [Microsoft Entra ID](https://learn.microsoft.com/en-us/training/modules/sharepoint-embedded-setup/5-exercise-setup-configure-sharepoint-embedded)
```

Comment:
don't use absolute links or locales when referencing pages in learn.Microsoft.com